components/core/src/os/users/windows.rs

```diff
@@ -12,22 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::env;
 use std::path::PathBuf;
 
+use habitat_win_users::account::*;
+
 extern "C" {
     pub fn GetUserTokenStatus() -> u32;
 }
 
-pub fn get_uid_by_name(owner: &str) -> Option<u32> {
-    unimplemented!();
+fn get_sid_by_name(name: &str) -> Option<String> {
+  match Account::from_name(name.to_string()) {
+    Some(acct) => Some(acct.sid.to_string()),
+    None => None
+  }
 }
 
-pub fn get_gid_by_name(group: &str) -> Option<u32> {
-    unimplemented!();
+pub fn get_uid_by_name(owner: &str) -> Option<String> {
+  get_sid_by_name(owner)
+}
+
+pub fn get_gid_by_name(group: &str) -> Option<String> {
+  get_sid_by_name(group)
 }
 
 pub fn get_current_username() -> Option<String> {
-    unimplemented!();
+    Some(env::var("USERNAME").unwrap())
```

Comment:
case will likely matter for any equality checks, so maybe force a downcase?

And what behavior would you like if `$env:USERNAME` is empty or not present? (maybe this is a place for a `match` or `unwrap_or_default`.